website_whatsapp/models/website.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Studio73 - Ioan Galan <ioan@studio73.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
```

Comment:
might be the file name needs to be switched with res_config_settings.py and website.py

Reply:
@bizzappdev done